File: Java/selectionsort.java
public class selectionsort
{
	public static void main(String[] args)
	{
		int n = 10;
		int[] a = new int[n];
		for (int i = 0; i < a.length ; i++) {
			a[i] = (int)(Math.random() * 100);
		}
		//selectioon sort
		for (int i = 0; i < a.length; i++) {
			int min = i;
			for (int j = i + 1; j < a.length ; j++) {
				if(a[j] < a[min])
				{
					min = j;
				}
			}
			int temp = a[i];
			a[i] = a[min];
			a[min] = temp;
		}
		for (int i = 0; i < a.length ; i++) {
			System.out.print(a[i] + " ");
		}	
	}
}

File: Python/selectionsort.py
import random

a = []
for i in range(10):
	a.append(int(random.random() * 100))
	
for i in range(len(a)):
	min = i
	for j in range(i + 1, len(a)):
		if a[j] < a[min]:
			min = j
	a[min], a[i] = a[i], a[min]

print(a)